recipes-qt/qt5/qtbase/0001-qsslsocket_openssl_symbols-Really-support-OpenSSL-3.patch

```diff
@@ -0,0 +1,105 @@
+From 8df5d7c30fe184fde7e25a4ebb2bc4994273eae4 Mon Sep 17 00:00:00 2001
+From: Mike Looijmans <mike.looijmans@topic.nl>
+Date: Fri, 4 Aug 2023 13:56:57 +0200
+Subject: [PATCH] qsslsocket_openssl_symbols: Really support OpenSSL 3.x
+
+Based on this:
+  https://gist.github.com/seyedmmousavi/b1f6681eb37f3edbe3dcabb6e89c5d43
```

Comment:
Missing Upstream-Status

Reply:
That'd ne "backport" since it's present in the commerceial version?